### ISSUE TITLE
Optionally not opening the browser?

### ISSUE THREAD

**Issue by Madd0g** [exah/alfred-hotel#2]
I understand that the workflow changes the url based on the modifier key and that going to the url starts the services. I'm wondering there's an easy way to avoid opening the browser? Most of my hotel stuff are services and not web-based and just need to be toggled on/off.

Would be really nice if there was a modifier to just toggle things on/off that also returned back into the list view. 

Thanks

**Comment by exah** [comment]
Yeah, I can add <kbd>Command</kbd> + <kbd>Enter</kbd> to start the app that not already running.

**Comment by exah** [comment]
Implemented in 0.0.4, you can update with `npm i -g alfred-hotel`

**Comment by Madd0g** [comment]
for some reason, after the upgrade the browser filter was empty ![](http://i.imgur.com/Zxnhc80.png)

I changed it to regex and put `^((?!start|stop))$` there and now it does exactly what I wanted.

EDIT: I did one more thing to solve my second problem - I added a trigger and an extra step that ![](http://i.imgur.com/fxr5cGx.png) shows the list again after start/stop.

Thanks! Very happy you took the time to make this
